components/mpas-ocean/src/shared/mpas_ocn_vel_tidal_potential.F

```diff
@@ -171,9 +174,9 @@ subroutine ocn_vel_tidal_potential_tend(ssh, surfacePressure, tend, err)!{{{
 
          potentialGrad = - gravity*invdcEdge* &
             (  tidalPotEta(cell2) - tidalPotEta(cell1) &
-             + betaSelfAttrLoad &
-              *((ssh(cell2) - ssh(cell1)) &
-                + rho0gInv*(surfacePressure(cell2) - surfacePressure(cell1))))
+             + (1.0_RKIND - ssh_sal_on) * betaSelfAttrLoad *( (ssh(cell2) - ssh(cell1)) & 
```

Comment:
In the acc directives at the beginning of this loop, please add ssh_sal to the list of present() variables

Reply:
Thanks for adding.